WORKSPACE

```diff
@@ -16,6 +16,13 @@ workspace(name = "io_opencensus_cpp")
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
+# Build rules for C++ projects.
+http_archive(
```

Comment:
For what it's worth, I would consider the fact that this dependency had to be added to your WORKSPACE a bug in the `abseil-cpp` repo. There doesn't appear to be anything stopping them from pulling this out to a `deps()` macro.

Reply:
Please correct me if I'm wrong but my understanding is that this will be required for all C++ projects after bazel 1.0(?) because rules like `cc_library` will move from native to skylark implementations.

Reply:
That may be the case for one hop. But suppose absl CPP requires `rules_cc` and your repo provides, say, a Golang tool. It's not intuitive to users of the Golang tool that they should have to pull in `rules_cc` despite never directly using a C++ rule.

This has already happened in the gRPC repo. We have to pull in a Golang-only tool used by Envoy even though we're only -- to my knowledge -- using some of their protos.

Reply:
Ditto. opencensus-cpp pulls in e.g. grpc-java, because opencensus-proto depends on it.

As for rules_cc specifically, I think opencensus-cpp will end up using them directly after bazel 1.0.